src/common/edi/common/util/core.clj

```diff
@@ -52,9 +52,11 @@
     (.array buffer)))
 
 
-(defn iso-datetime->java-util-date [^String v]
-  (->> v
-       (.parse DateTimeFormatter/ISO_INSTANT)
+(defn parse-java-util-date [^String v]
+  (->> (cond-> v
+               (= \space (nth v 10)) (#(str (subs % 0 10) "T" (subs % 11)))
+               (re-matches #".*[+-]\d\d$" (subs v 19)) (str ":00"))
```

Comment:
Prefer not to use RegEx here - there're not only whole numbers as TZ and also magic numbers are looking ugly and annoying. Such way of describing the function is unsupportable.